release/lts/nanoserver1809/meta.json

```
@@ -1,10 +1,20 @@
 {
-    "IsLinux" : false,
+    "IsLinux": false,
     "PackageFormat": "PowerShell-${PS_VERSION}-win-x64.zip",
     "osVersion": "Nano Server, version ${fromTag}",
     "shortTags": [
-        {"Tag": "1809"},
-        {"Tag": "1903"}
+        {
+            "Tag": "1809"
+        },
+        {
+            "Tag": "1903"
+        },
+        {
+            "Tag": "1909"
+        },
+        {
+            "Tag": "2004"
+        }
     ],
     "tagTemplates": [
         "lts-nanoserver-#shorttag#"
```

Where getting rid of getLatestTag.ps1, you should be able to delete it from the folder, with this change to the meta.json.
```suggestion
        "lts-nanoserver-#shorttag#",
        "lts-nanoserver-2004-#fulltag#"
```

Uhg, this won't work unless we separate each image into it's own file.

Not needed for this PR.